app/views/hosts/_form.html.erb

```diff
@@ -85,6 +85,8 @@
       <%= render_tab_content_for(:main_tabs, :subject => @host) %>
 
       <%= f.hidden_field :managed %>
+      <%= f.hidden_field :uuid %>
+      <%= f.hidden_field :compute_resource_id if @host.new_record? && @host.uuid.present? %>
```

Comment:
can you explain a bit more about how its used? I'm afraid its unclear whats needed for import vs new vs edit. (maybe code comment would be best).